js/src/ui/Form/AddressSelect/addressSelect.css

```diff
@@ -14,53 +14,254 @@
 /* You should have received a copy of the GNU General Public License
 /* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 */
-.account {
-  padding: 0.25em 0;
-  display: flex;
-  align-items: center;
+
+.input {
+  box-sizing: border-box;
+  appearance: textfield;
+  width: 100%;
+  padding: 0;
+  border: none;
+  background: transparent;
+
+  transition-property: font-size, padding;
+  transition-duration: 0.5s;
+  transition-timing-function: cubic-bezier(0.7,0,0.3,1);
+
+  color: white;
+  font-family: inherit;
+  font-size: 2em;
+
+  &:focus {
+    outline: none;
+  }
+
+  &::placeholder {
+    color: #a2a2a2;
+  }
 }
 
-.name {
-  height: 32px;
-  line-height: 32px;
-  display: inline-block;
-  vertical-align: top;
-  text-transform: uppercase;
-  padding: 0 0 0 1em;
+.parityBackground {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  opacity: 0.25;
+  z-index: -1;
 }
 
-.balance {
-  color: #aaa;
-  padding-left: 1em;
+.inputAddress {
+  position: relative;
+
+  &:hover, *:hover {
+    cursor: pointer !important;
+  }
 }
 
-.image {
-  display: inline-block;
-  height: 32px;
-  width: 32px;
-  margin: 0;
-  z-index: 10;
+.main {
+  position: relative;
+  left: 0;
+
+  &:focus {
+    outline: none;
+  }
 }
 
-.icon {
+.closeIcon {
   position: absolute;
-  left: 0;
-  top: 35px;
+  top: 0.5rem;
+  right: 1rem;
+  font-size: 4em;
+
+  height: 48px;
+  width: 48px;
 
-  &.noLabel {
-    top: 11px;
+  &:hover {
+    cursor: pointer;
   }
 }
 
-.paddedInput input {
-  padding-left: 46px !important;
+.label {
+  margin: 1rem 2rem 0.25em;
+  color: rgba(255, 255, 255, 0.498039);
 }
 
-.container {
+.underline {
   position: relative;
+  margin: 0 9rem 0 2rem;
 }
 
-.menuItem {
-  min-height: 0 !important;
-  line-height: inherit !important;
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 80px;
+
+  transform-origin: 100% 0;
+  transition-property: height, width, top, left, background-color, z-index;
+  transition-duration: 0.5s;
+  transition-timing-function: ease-out;
+
+  background-color: rgba(0, 0, 0, 0.0);
+  z-index: -9999;
+
+  display: flex;
+
+  * {
+    min-width: 0;
+  }
+
+  &.expanded {
+    height: 100vh;
+
+    background-color: rgba(0, 0, 0, 0.85);
+    z-index: 9999;
+
+    .input {
+      font-size: 3em;
+      padding: 0 9rem 0.5em 2rem;
+      display: block;
+
+      padding-right: 6rem;
+    }
+  }
+}
+
+.empty {
+  font-size: 1.5em;
+}
+
+.inputContainer {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+}
+
+.categories {
+  flex: 1;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+
+  margin: 2rem 2rem 0;
+
+  > * {
+    flex: 1;
+  }
+}
+
+.category {
+  display: flex;
+  flex-direction: column;
+  margin: 0 0.5em;
+  max-width: 35em;
```

Comment:
It could but having too large components (here _cards_) make them look strange. I would rather a max-width than something taking up the whole width with a huge gap of nothing in the middle (since there are informations on the left and on the right of the card).